### ISSUE TITLE
MY_ACCESS_TOKEN gives 401 if collected under Firefox

### ISSUE THREAD

**Issue by stereotypy** [mmrobins/creditkarma_export_transactions#2]
Hey, just wanted to add this here. I kept getting the help message and was thinking somehow my ENV variable wasn't taking, so I put it directly in the code. It was unclear to me that the help message also shows up under a 401 as the error message looks similar to what's being printed in help itself.

I stopped getting an issue once I got the token from Chromium instead (although I believe I ran into another issue because it wanted me to verify by text message and I got blocked by accidentally requesting too many resends, will check back about this if it doesn't resolve).

**Comment by mmrobins** [comment]
I updated the script to hopefully make the error message clear by spitting out after the help.  Also tried to make the error that spits out something you could copy paste into an issue so it's easier to debug.  I'm assuming you got this working eventually once you did the text message verification step.  If not, feel free to reopen and paste in your error message.

FWIW the token I got from firefox (my primary browser) works the same for me as the one I got from Chrome.  I'm GUESSING you copied different portions depending on which dev tools you used.

**Comment by dylankb** [comment]
Just came here to say that I ran into the same issue where the token fetched from Firefox (I tried many times) returned a 401 but the token fetched from Chrome worked on the first try. Both requests appeared to be identical and were from the  /networth/transactions page. Doesn't really make much sense to me why that would be 🤷 .
